mobx_codegen/test/data/valid_store_with_ui_types_input.dart

```diff
@@ -0,0 +1,36 @@
+library generator_sample;
+
+import 'dart:ui';
+
+import 'package:mobx/mobx.dart';
+
+part 'generator_sample.g.dart';
```

Comment:
I think we can remove this line and the test should still work. Can we try that.

Reply:
Addressed in previous comment.